<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrakerSelectDialog</class>
 <widget class="QDialog" name="TrakerSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker Select</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QRadioButton{

font-weight: bold;
font-size: 15px;

}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="radioButton_ct_tracker">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>59</y>
      <width>167</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Compress Tracker</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_particle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>43</y>
      <width>158</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Particle Filter</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_mil_boost">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>75</y>
      <width>203</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Online MIL boosting </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_apply">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
